front/app/containers/ProjectsShowPage/index.tsx

```diff
@@ -120,6 +154,26 @@ const ProjectsShowPage = memo<Props>(({ project, scrollToEventId }) => {
   } else if (projectNotFound) {
     content = <ProjectNotFound />;
   } else if (projectId && processType) {
+    let phaseParticipationMethod;
+
+    if (!isNilOrError(phases)) {
+      if (phaseIdUrl) {
+        const phase = getPhase(phaseIdUrl, phases);
+        if (!isNilOrError(phase)) {
+          phaseParticipationMethod = phase.attributes.participation_method;
+        }
+      } else {
+        phaseParticipationMethod =
+          getCurrentPhase(phases)?.attributes.participation_method;
+      }
+    }
+
+    const config = getMethodConfig(
+      phaseParticipationMethod
```

Comment:
I suspect that based on the logic, the `phaseParticipationMethod` could be null but we should be able to get this once we add types everywhere else.